SimplifiedTests/Date+NYPLAdditionsTests.swift

```diff
@@ -37,49 +37,4 @@ class Date_NYPLAdditionsTests: XCTestCase {
     XCTAssertEqual(iOS10Date?.utcComponents().month, 6)
     XCTAssertEqual(iOS10Date?.utcComponents().day, 2)
   }
-
-  func testInvalidRFC3339Date() {
```

Comment:
these were already moved, correct? sorry, I forget

Reply:
Yeah, these are in NYPLAudiobookToolkit repo